packages/components/src/internal/app/utils.ts

```diff
@@ -227,6 +227,14 @@ export function isAssayEnabled(moduleContext?: ModuleContext): boolean {
     );
 }
 
+export function isComplianceEnabled(moduleContext?: ModuleContext): boolean {
```

Comment:
As mentioned in the other PR, perhaps this wants a different name, unless we think we'll use it for actual compliance features.

Reply:
Done.